### PR TITLE
Change default EBS volume type from gp2 to gp3

### PR DESCRIPTION
**Pull request by francesco-giordano**
### Description of changes
* Change default EBS volume type from gp2 to gp3

### Test
* Automated tests: test_ebs and create_ami
* Manual tests: created an ami an then created a cluster

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ x.x.x
 - Show `requested_value` and `current_value` values in the change set when adding or removing a section.
 - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based
   scheduling in Slurm.
+- Change default EBS disks volume types form gp2 to gp3
```
Comment (lukeseawalker) [inline CHANGELOG.md]: typo `form` -> `from`
also, you can remove `disks` and add that the change is both for root volume and additional volumes

Comment (francesco-giordano) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ x.x.x
 - Show `requested_value` and `current_value` values in the change set when adding or removing a section.
 - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based
   scheduling in Slurm.
+- Change default EBS volume types from gp2 to gp3 in both the root and additional volumes
```
Comment (enrico-usai) [inline CHANGELOG.md]: Missing final period.

File: tests/integration-tests/tests/storage/test_ebs.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def test_ebs_snapshot(
 
     mount_dir = "/" + mount_dir
     scheduler_commands = scheduler_commands_factory(remote_command_executor)
-    _test_ebs_correctly_mounted(remote_command_executor, mount_dir, volume_size="9.8")
+    _test_ebs_correctly_mounted(remote_command_executor, mount_dir, volume_size="9.7")
```
Comment (enrico-usai) [inline tests/integration-tests/tests/storage/test_ebs.py:75]: Why this change?

File: CHANGELOG.md (hunk 2 above)
Comment (francesco-giordano) [reply CHANGELOG.md]: Done

**Review by enrico-usai — DISMISSED** [review]

**Review by enrico-usai — APPROVED** [review]
